rental.rb

```diff
@@ -0,0 +1,8 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, _person)
+    @date = date
+    @book = book
+  end
+end
```

Comment:
For the rental class, the rental should add itself to the @book instance achieving the 'belongs-to' association😉😉. Also, The rental should also add itself to the @person instance so as to fulfill the 'belongs-to' association😉.

<img width="1009" alt="image" src="https://github.com/Mahamadou-Nouridine/oop-school-library/assets/64643995/58ce16b6-6818-4079-a777-a2ac386ee20b">